packages/crud/src/RecordCreate.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
 import { MutationResult, MutationFn } from 'react-apollo';
-import { TableConsumer } from '@8base/table-schema-provider';
+import { TableConsumer, ITableConsumerRenderProps } from '@8base/table-schema-provider';
 import { TableSchema, SDKError, ERROR_CODES, PACKAGES } from '@8base/utils';
 
 import { RecordCrud } from './RecordCrud';
 
 interface IChildrenPropObject {
-  tableMetaResult: TableSchema | null;
+  tableMeta: TableSchema | null;
```

Comment:
@ej9x maybe we could name it as `tableSchema`? @vorobeez what do u think about it?

Reply:
fixed